apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs

```diff
@@ -61,11 +61,20 @@ public DateTime ToDateTime()
         /// <summary>
         /// Compares the versions to another.
         /// </summary>
+        /// <param name="obj">The other version to compare this to.</param>
+        /// <returns>
+        /// A negative value if this is less than the other version, a positive value if this is greater, and zero if they are equal.
+        /// </returns>
+        public int CompareTo(object obj) => Compare(this, obj as BigtableVersion?);
```

Comment:
I think this should throw ArgumentException if compared with a non-BigtableVersion.

Reply:
Done